prow/gerrit/reporter/reporter.go

```diff
@@ -206,26 +204,22 @@ func (c *Client) Report(pj *v1.ProwJob) ([]*v1.ProwJob, error) {
 		logger.Warn("Tried to report empty or aborted jobs.")
 		return nil, nil
 	}
-
-	vote := client.LBTM
-	if report.success == report.total {
-		vote = client.LGTM
+	var reviewLabels map[string]string
+	if reportLabel != "" {
+		vote := client.LBTM
+		if report.success == report.total {
+			vote = client.LGTM
+		}
+		reviewLabels = map[string]string{reportLabel: vote}
 	}
-	reviewLabels := map[string]string{reportLabel: vote}
 
 	logger.Infof("Reporting to instance %s on id %s with message %s", gerritInstance, gerritID, message)
 	if err := c.gc.SetReview(gerritInstance, gerritID, gerritRevision, message, reviewLabels); err != nil {
-		logger.WithError(err).Errorf("fail to set review with %s label on change ID %s", reportLabel, gerritID)
-
-		// possibly don't have label permissions, try without labels
-		message := fmt.Sprintf("[NOTICE]: Prow Bot cannot access %s label!\n%s", reportLabel, message)
```

Comment:
If we did try to report to a label and failed, we should still retry without the label so that the user gets some indication that the job completed and why the label wasn't set.

Reply:
makes sense, added back.